app.rb

```diff
@@ -0,0 +1,137 @@
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
```

Comment:
![image](https://github.com/noelfoka/library-school/assets/93039359/999689ff-e1d4-44d1-901c-0c9e61d60af5)

- [ ] It would be nice if you fix this problem in the person class and replace the colon with equal sign only in the highlighted place 👍 to fix the issue from one side 😊

Reply:
This change has been done well